api/resources/views/job-form/edit.blade.php

```diff
@@ -0,0 +1,30 @@
+@extends('layouts.app')
+
+@section('content')
+
+    <div class="container mt-4">
+        @include('errors')
+        <form action="{{ route('job-forms.update', $jobForm->id) }}" method="post">
+            @csrf
+            @method('put')
+
+            <div class="form-group">
+                <label for="name">Form Name</label>
+                <input type="text" class="form-control form-control" id="name" name="name" placeholder="Enter form name" value="{{$jobForm->name}}">
+            </div>
+
+            @foreach($fields as $field)
+                <div class="form-group">
+                    <label style="text-transform: capitalize" for="select-{{$field}}">{{ implode(" ", explode('_', $field)) }} field</label>
```

Comment:
Could've used bootstrap's text-capitalize class instead of an inline style tag.

Reply:
yeahh

Reply:
done